### ISSUE TITLE
Heroku RAM exceeded issue when mirroring multiple tasks at once

### ISSUE THREAD

**Issue by tiararosebiezetta** [anasty17/mirror-leech-telegram-bot#132]
I usually use this bot from this repo for mirroring a task one at a time, I never found out about this issue.

But today, I tried to mirror two magnet links at once and when one of the task was completely uploaded, the bot was stuck. I tried it in my another bot, it was the same. And then I tried it with my friend's bots with the latest commit, and they were the same.

The log only gave this:
`Error R14 (Memory quota exceeded)`

There's nothing else mentioned, when the bot is stuck, the log doesn't show anything and I need to restart the dyno to fix this, but the problem will be still there if I try to mirror more than one tasks.

I don't know about how that was in older commits so I can't be sure what is the cause if it's from the repo. It can also be that heroku gives a less powerful server for free tier apps, I just want to make sure of this. It can also be something wrong in my part.

Thank you 🙏

**Comment by anasty17** [comment]
@tiararosebiezetta Yeah there is a problem... will push fix after minutes... thanks for reporting

**Comment by anasty17** [comment]
Fixed refork

**Comment by tiararosebiezetta** [comment]
Thank you so much 👍

**Comment by anasty17** [comment]
@tiararosebiezetta U wlc :)
